tests/ui/traits/const-traits/staged-api.rs

```diff
@@ -16,10 +16,8 @@ use staged_api::*;
 pub struct Foo;
 
 #[stable(feature = "rust1", since = "1.0.0")]
-#[cfg_attr(unstable, rustc_const_unstable(feature = "local_feature", issue = "none"))]
-#[cfg_attr(stable, rustc_const_stable(feature = "local_feature", since = "1.0.0"))]
+#[rustc_const_unstable(feature = "local_feature", issue = "none")] // FIXME this attribute makes no sense?
```

Comment:
Something seems to be requiring this attribute so I can't remove it... the trait solver doesn't take into account stability, right? So this doesn't really make sense?

Reply:
Trait solver definitely doesn't take into account stability. I think you can probably just remove the code that requires that const stability is marked on impls, since yes, it both:
1. doesn't make sense
2. can't even be anything other than `rustc_const_unstable` today.

Reply:
I'd rather not touch the stability attribute code if I can avoid it.^^ That code is such a mess... and it would likely conflict with https://github.com/rust-lang/rust/pull/132541.